requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Install requirements
 numpy==1.21.6; python_version < '3.10'  # support numpy.typing
-oldest-supported-numpy; python_version >= '3.10'
+numpy==2.1.0; python_version >= '3.10'
```

Comment:
`oldest-supported-numpy` is deprecated. And now that we don't require NumPy as build time we can be a lot less finicky with it.